app/models/page.rb

```diff
@@ -45,7 +45,7 @@ def submit
 
   def load_from_session(session, keys)
     keys.each do |key|
-      self.load("#{key}": session.dig(:page, key) || send(key.to_sym))
+      self.load("#{key}": session.dig(:page, key.to_sym) || send(key.to_sym))
```

Comment:
🤔  I was in two minds about creating a new local variable just above the the call to `each` which would store a version of session but with its keys symbolized like https://github.com/alphagov/forms-admin/blob/main/app/controllers/pages_controller.rb#L122 so if the session passed in does have a mixture of strings/symbols for some odd reason then the values can still be found.

Any thoughts?

Reply:
What about using `ActiveSupport::HashWithIndifferentAccess`? 

https://api.rubyonrails.org/classes/ActiveSupport/HashWithIndifferentAccess.html

Reply:
Thats a good option too. Leave this with me